components/helm-broker/internal/broker/deprovision.go

```diff
@@ -113,11 +115,15 @@ func (svc *deprovisionService) doAsync(ctx context.Context, iID internal.Instanc
 func (svc *deprovisionService) do(ctx context.Context, iID internal.InstanceID, opID internal.OperationID, releaseName internal.ReleaseName) {
 
 	fDo := func() error {
-		if err := svc.helmDeleter.Delete(releaseName); err != nil {
+		err := svc.helmDeleter.Delete(releaseName)
+		switch {
+		case err == nil:
+		case strings.Contains(err.Error(), helmErrors.ErrReleaseNotFound(string(releaseName)).Error()):
```

Comment:
i know that looks strange but in that way they handle errors in helm e.g: https://github.com/helm/helm/blob/master/cmd/helm/upgrade.go#L222